fonts/stylesheet.css

```diff
@@ -0,0 +1,44 @@
+/*! Generated by Font Squirrel (https://www.fontsquirrel.com) on September 25, 2020 */
+
+
+
+@font-face {
+    font-family: 'pt_serifbold';
+    src: url('ptserif-bold-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/html-css-2-0/pullRequest?prid=6240894)